examples/UNet/generators.py

```diff
@@ -21,6 +21,22 @@ def batch_generator(data, target, BATCH_SIZE, shuffle=False):
         if ctr >= data.shape[0]:
             ctr -= data.shape[0]
 
+
+def center_crop_generator(generator, output_size):
+    '''
+    yields center crop of size output_size (may be 1d or 2d) from data and seg
+    '''
+    if type(output_size) not in (tuple, list):
+        center_crop = [output_size, output_size]
+    elif len(output_size) == 2:
+        center_crop = list(output_size)
+    else:
+        raise ValueError("invalid output_size")
```

Comment:
Hint: You can use `center_crop = lasagne.utils.as_tuple(output_size, 2, int)`. This will take care of everything.